bin/strap.sh

```diff
@@ -213,6 +217,23 @@ if ! [ -f "$HOME/.ssh/id_rsa" ]; then
   PUBLIC_KEY="$(cat $HOME/.ssh/id_rsa.pub)"
   POST_BODY="{\"title\":\"MacOSX Key - strap\",\"key\":\"$PUBLIC_KEY\"}"
   curl $Q -H "Content-Type: application/json" -H "Authorization: token $STRAP_GITHUB_TOKEN" -X POST -d "$POST_BODY" https://api.github.com/user/keys
+
+  log "checking for known_hosts file"
+  if ! [ -f "$HOME/.ssh/known_hosts"]; then
+    log "creating known_hosts file"
+    touch ~/.ssh/known_hosts
+    chmod 644 ~/.ssh/known_hosts
+  else
+    log 'found known_hosts file at ~/.ssh/known_hosts'
+  fi
+
+  log "checking for github.com as a known host"
+  if [ `ssh-keygen -H -F githubr.com | wc -l` = 0 ]; then
```

Comment:
is githubr.com correct?  Or should it be github.com?

Reply:
it should be github.com, i was testing and forgot to change it back